.github/workflows/lint.yml

```diff
@@ -18,4 +18,5 @@ jobs:
           node-version: 20
           cache: "pnpm"
       - run: pnpm install
+      - run: pnpm build
```

Comment:
I'm guessing this worked before because we used `bundler` resolution which basically tells typescript not to care so much about unresolved imports